package.json

```diff
@@ -10,6 +10,11 @@
     "base64",
     "gzip"
   ],
+  "dependencies": {
+    "stream":"~0.0.2",
+    "util": "~0.10.3",
+    "string_decoder": "~0.10.31"
```

Comment:
Wait... what? These are native modules, not npm packages.

Can you explain what problems you're seeing that's prompted this PR?